### PR TITLE
Add instructions for VS Code

### PR DESCRIPTION
**Pull request by emceeaich**

### CODE REVIEW

**Comment by maebert** [comment]
Awesome, thanks! Could you rebase that against `2.0-rc-1`?

**Comment by Hatoris** [comment]
I just setting mine to run on windows machine, you can add it to your commit : 

```rst
To set `Visual Studo Code <https://code.visualstudio.com/>`_ as your editor on windows, edit ``.jrnl_config`` like this: 

.. code-block:: javascript
    "editor" : "C:\\Program Files\\Microsoft VS Code\\Code.exe -n -w"
  
The -n argument tells VS Code to open new session of VS Code instead of restoring the previous session, it's necessary to close this session to handing back control to jnrl.
The -w argument tells VS Code to wait for files to be written out before handing back control to jrnl. 
```

**Comment by micahellison** [comment]
Hello! We're just getting the jrnl repo going again and are working on getting a release together. Would you be willing to rebase against the current master branch? Thanks.

**Comment by emceeaich** [comment]
Yes, let me look,a this this afternoon.

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Optionally, your journal can be encrypted using the [256-bit AES](http://en.wiki
 
 ### Why keep a journal?
 
-Journals aren't only for 13-year old girls and people who have too much time on their summer vacation. A journal helps you to keep track of the things you get done and how you did them. Your imagination may be limitless, but your memory isn't. For personal use, make it a good habit to write at least 20 words a day. Just to reflect what made this day special, why you haven't wasted it. For professional use, consider a text-based journal to be the perfect complement to your GTD todo list - a documentation of what and how you've done it.
+Journals aren't just for angsty teenagers and people who have too much time on their summer vacation. A journal helps you to keep track of the things you get done and how you did them. Your imagination may be limitless, but your memory isn't. For personal use, make it a good habit to write at least 20 words a day. Just to reflect what made this day special, why you haven't wasted it. For professional use, consider a text-based journal to be the perfect complement to your GTD todo list - a documentation of what and how you've done it.
```
Comment (emceeaich) [inline README.md]: s/Journals aren't just for angsty teenagers and people who have too much time on their summer vacation./Journals are for everyone/

File: docs/overview.rst [hunk 2]
```diff
@@ -15,7 +15,7 @@ Optionally, your journal can be encrypted using the `256-bit AES <http://en.wiki
 Why keep a journal?
 -------------------
 
-Journals aren't only for 13-year old girls and people who have too much time on their summer vacation. A journal helps you to keep track of the things you get done and how you did them. Your imagination may be limitless, but your memory isn't.
+Journals aren't just for angsty teenagers and people who have too much time on their summer vacation. A journal helps you to keep track of the things you get done and how you did them. Your imagination may be limitless, but your memory isn't.
```
Comment (emceeaich) [inline docs/overview.rst]: s/Journals aren't just for angsty teenagers and people who have too much time on their summer vacation/Journals are for everyone/

File: setup.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 Why keep a journal?
 ```````````````````
 
-Journals aren't only for 13-year old girls and people who have too much time on their summer vacation. A journal helps you to keep track of the things you get done and how you did them. Your imagination may be limitless, but your memory isn't. For personal use, make it a good habit to write at least 20 words a day. Just to reflect what made this day special, why you haven't wasted it. For professional use, consider a text-based journal to be the perfect complement to your GTD todo list - a documentation of what and how you've done it.
+Journals aren't just for angsty teenagers and people who have too much time on their summer vacation. A journal helps you to keep track of the things you get done and how you did them. Your imagination may be limitless, but your memory isn't. For personal use, make it a good habit to write at least 20 words a day. Just to reflect what made this day special, why you haven't wasted it. For professional use, consider a text-based journal to be the perfect complement to your GTD todo list - a documentation of what and how you've done it.
```
Comment (emceeaich) [inline setup.py]: s/Journals aren't just for angsty teenagers and people who have too much time on their summer vacation/Journals are for everyone/

**Comment by emceeaich** [comment]
Thanks for your patience while I got the conflicts resolved.

**Comment by emceeaich** [comment]
Let me try a local checkout of my branch and see what's breaking.

**Comment by maebert** [comment]
Hey @emceeaich, could you have a look at #611 (and the branch underneath)? I'm suggesting moving to markdown docs to make editing easier, would love to see your changes in there!

**Comment by emceeaich** [comment]
Agreed. 

Should I cherry pick my changes into that new branch and close this PR?

**Comment by maebert** [comment]
That would be great, yes!

**Comment by emceeaich** [comment]
No worries!

File: docs/overview.rst (hunk 2 above)
Comment (wren) [reply docs/overview.rst]: I like this much better

**Pull request closed without merging**